cmd/machine-config-daemon/start.go

```diff
@@ -202,7 +207,13 @@ func runStartCmd(_ *cobra.Command, _ []string) {
 		klog.Fatalf("Could not get FG, timed out: %v", err)
 	}
 
-	if err := dn.Run(stopCh, exitCh); err != nil {
+	if err := dn.Run(stopCh, exitCh, errCh); err != nil {
+		if errors.Is(err, daemon.ErrAuxiliary) {
+			dn.CancelSIGTERM()
+			dn.Close()
+			os.Exit(255)
```

Comment:
I... think verify is complaining that this won't run the defer cancel() in the function above?

Reply:
yeah, I moved this here because where it used to be was ALSO an issue. I might just duplicate the cancel() call in here and add a nolint